### ISSUE TITLE
Display images attached to message

### ISSUE THREAD

**Issue by ivy-rew** [EionRobb/purple-teams#3]
I'm just wondering if it's a difficult task to show images which were sent in chats?
Right now it looks as if I get no evidence that an image has been sent to illustrate a message. I think to start with any kind of notice on the image would help, maybe you can easily add a link to the image to see it in the browser? 
![teamsPictures](https://user-images.githubusercontent.com/15085693/163960332-21bf1db9-3191-4a45-a6d4-7cfbf5e9de37.png)

**Comment by EionRobb** [comment]
That looks better than what I get:
![image](https://user-images.githubusercontent.com/1063865/164087924-0da976d0-f9b3-4dbf-aa17-1f98bfe2079c.png)

Also, check you're not using the Conversation Colours plugin (in Tools->Plugins) as this will eat any inline images (not that they'll show at the moment until I process the <img> urls)

**Comment by ivy-rew** [comment]
Thanks for your response @EionRobb .
I think my screenshot/description was misleading. Actually, the screenshot was produced from picture that I've sent, via teams client. Therefore not really a required necessity to view that file on another chat client ... As I do not want to keep the real teams client and pidgin in parallel for a long time.

Nevertheless, I can confirm that the 'red-x' icon appears if another user sends and image to me. So, I have a hint that a picture was sent, but unfortunately I can't view it without switching to real teams client.
Also, I've checked my plugins 'conversation colours' is not enabled.

So ultimately, It would be cool to have a live preview or a link to see pictures that we're sent in chats by other participants.

**Comment by EionRobb** [comment]
That latest commit kinda does the trick. Annoying that emoji also try to come through as images. Need to chop them out

**Comment by ivy-rew** [comment]
Yes, indeed that works. Great job @EionRobb  :rocket: 
![discussWithImages](https://user-images.githubusercontent.com/15085693/164388456-45b0f0a1-8528-4c10-a93d-ecbc37999006.png)

**Comment by jebej** [comment]
Hello, thanks for the plugin. On my end, when someone sends an image I just get a link that says "Click here to view full version". Is that expected?

**Comment by EionRobb** [comment]
> Hello, thanks for the plugin. On my end, when someone sends an image I just get a link that says "Click here to view full version". Is that expected?

@jebej Could you open a new issue for that?  Also, can you check that you're not using the "Conversation colors" plugin, as that's known to eat inline images in Pidgin
